apps/AEPSampleAppNewArchEnabled/README.md

```diff
@@ -0,0 +1,50 @@
+# Welcome to your Expo app 👋
+
+This is an [Expo](https://expo.dev) project created with [`create-expo-app`](https://www.npmjs.com/package/create-expo-app).
+
+## Get started
+
+1. Install dependencies
+
+   ```bash
+   npm install
+   ```
+
+2. Start the app
+
+   ```bash
+    npx expo start
+   ```
+
+In the output, you'll find options to open the app in a
+
+- [development build](https://docs.expo.dev/develop/development-builds/introduction/)
+- [Android emulator](https://docs.expo.dev/workflow/android-studio-emulator/)
+- [iOS simulator](https://docs.expo.dev/workflow/ios-simulator/)
+- [Expo Go](https://expo.dev/go), a limited sandbox for trying out app development with Expo
+
```

Comment:
I am getting error when run the android app

ERROR  Warning: Error: [Reanimated] Mismatch between JavaScript part and native part of Reanimated (3.15.5 vs 3.10.1).
    See `https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting#mismatch-between-javascript-part-and-native-part-of-reanimated` for more details., js engine: hermes 
    at ContextNavigator (http://192.168.0.104:8081/node_modules/expo-router/entry.bundle//&platform=android&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.routerRoot=app:146287:24)
    at ExpoRoot (http://192.168.0.104:8081/node_modules/expo-router/entry.bundle//&platform=android&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.bytecode=true&transform.routerRoot=app:146243:28)

Do you see the same error?

Reply:
---

### Steps to Run This Branch:

1. In your root folder, run:
   ```
   yarn install
   ```

2. Next, build the project by running:
   ```
   yarn run build
   ```

3. Navigate to the `apps` folder:
   ```
   cd apps/AEPSampleAppNewArchEnabled
   ```

4. Install dependencies by running:
   ```
   yarn install
   ```

5. Finally, run the Android build:
   ```
   yarn android or yarn ios 
   ```

---

try these steps for running this project as we are using expo prebuild for this project